tests/system/robot/chromeTests.py

```diff
@@ -48,6 +48,64 @@ def checkbox_labelled_by_inner_element():
 	)
 
 
+def test_aria_details():
+	_chrome.prepareChrome(
+		"""
+		<div>
+			<p>The word <mark aria-details="cat-details">cat</mark> has a comment tied to it.</p>
+			<div id="cat-details" role="comment">
+				Cats go woof BTW<br>&mdash;Jonathon Commentor
+				<div role="comment">
+				No they don't<br>&mdash;Zara
+				</div>
+				<div role="form">
+				<textarea cols="80" placeholder="Add reply..."></textarea>
+				<input type="submit">
+				</div>
+			</div>
+		</div>
+		"""
+	)
+	actualSpeech = _chrome.getSpeechAfterKey('downArrow')
+	# _asserts.strings_match(
+	# 	actualSpeech,
+	# 	"The word  marked content  has details  cat  out of marked content  has a comment tied to it."
+	# )

```

Comment:
Is this here for a reason? I assume just old code to be deleted?

Reply:
yes this is to be introduced when we can introduce the discovery